quiche/src/recovery/cubic.rs

```diff
@@ -787,319 +764,6 @@ mod tests {
         );
     }
 
-    #[test]
-    fn cubic_hystart_css_to_ss() {
```

Comment:
What does deleting these tests really mean?

Reply:
The tests were not testing an actual functionality as hystart is in fact called for every sent packet and not as performed in those tests at an arbitrary point in time. Because the new code folds hystart into cubic it is impossible to fake this non-existent behavior. It is easy to change the test to match the actual behavior, but then it would be pretty moot.

Reply:
I brought the tests back.